spinal_levels/README.md

```diff
@@ -0,0 +1,78 @@
+### PAM50 Levels
+
+These level files are a slightly modified copy of the [level files](https://github.com/PhillipsLab/pam50/tree/main/Spinal%20Cord%20Levels%20NIfTI) produced by the [Phillips Lab](https://github.com/PhillipsLab): 
+
+Modifications include:
+
+- Change data type from float64 to float32
+- Copy header from current PAM50/spinal_levels 
+- Rename files
+
+To reproduce the modified files, please run `git checkout e854bbad9ab550fd93acabeaf43c97cf66b3a4e5`, then run the following script in your terminal:
+
+```bash
+#!/bin/bash
+#
+# Process Phillips Lab PAM50 spinal levels to match existing PAM50 conventions.
+#
+# Usage:
+#   ./process_spinal_levels.sh
+# 1. Clone https://github.com/spinalcordtoolbox/PAM50
+# 2. Checkout commit e854bbad9ab550fd93acabeaf43c97cf66b3a4e5
+# 3. Run inside /PAM50/spinal_levels_PhillipsLab/
+# Authors: Sandrine Bédard, Joshua Newton
+
+set -x
+# Immediately exit if error
+set -e -o pipefail
+
+# Exit if user presses CTRL+C (Linux) or CMD+C (OSX)
+trap "echo Caught Keyboard Interrupt within script. Exiting now.; exit" INT
+
+start=`date +%s`
+
+# Add missing info to the `info_label.txt` file to account for newly-added levels
+file_info_label=$(realpath "../spinal_levels/info_label.txt")
+extra_spinal_levels="20, Spinal level L1, spinal_level_21.nii.gz
+21, Spinal level L2, spinal_level_22.nii.gz
+22, Spinal level L3, spinal_level_23.nii.gz
+23, Spinal level L4, spinal_level_24.nii.gz
+24, Spinal level L5, spinal_level_25.nii.gz"
+if [[ $(tail -c 23 "$file_info_label") == "spinal_level_20.nii.gz" ]]
+then
+    echo "$extra_spinal_levels" >> "$file_info_label"
+fi
+
+# Retrieve input params
+PATH_IN=$PWD
+PATH_OUT="$PATH_IN/spinal_levels_processed"
+for FILE in *.nii.gz; do
+    file=${FILE/%".nii.gz"}
+    echo $file
+    mkdir -p $PATH_OUT/${file}_processed
+    rsync -avzh $FILE $PATH_OUT/${file}_processed
+    cd $PATH_OUT/${file}_processed
+    # Change file type 
+    sct_image -i ${file}.nii.gz -type float32 -o ${file}_float32.nii.gz
+    file="${file}_float32"
+    # Copy header of SCT PAM50 template
+    sct_image -i $SCT_DIR/data/PAM50/spinal_levels/spinal_level_02.nii.gz -copy-header $file.nii.gz -o ${file}_header.nii.gz
+    file="${file}_header"
```

Comment:
## Header issue (image resolution)

> > The world coordinate between the source (Phillips lab) and PAM50, for the same voxel location, is drastically different. I guess this this is the reason for the qform copy (see example below).
> >
> > ![image](https://user-images.githubusercontent.com/2482071/218617926-930ccce0-1ea4-43cf-9467-e1b152bcc0ac.png)
> > ![image](https://user-images.githubusercontent.com/2482071/218618035-9fb90988-ba71-4c06-80ba-f5f07f12d5e6.png)
>
> Regarding the differences in world coordinates and resampling, I wasn't sure how to proceed.
> 
> The dimensions and resolution of:
> 
>   * New spinal levels files are : `141x141x991` and  `1x1x1`
>   * Old spinal levels files are: `141x141x991` and `0.5x0.5x0.5`
> 
> If we do resampling to 0.5 mm isotropic, the matrix size doubles, wich doesn't match anymore the PAM50 space.
> 
> I thought of doing registration with identity, however, since the world coordinates differ, it dosn't bring the spinal levels in the PAM50 space. How I overcame this issue is by copying the image header (for qform) and this also solves the resolution problem. I am not sure this is the best way to overcome this however...

Reply:
>     How I overcame this issue is by copying the image header (for qform) and this also solves the resolution problem. I am not sure this is the best way to overcome this however...

Following discussion in the previous SCT meeting, we came to the conclusion that this was a suitable approach to take.